src/dsl/journey.ts

```diff
@@ -25,22 +25,21 @@
 
 import { Browser, Page, BrowserContext, CDPSession } from 'playwright-chromium';
 import { Step } from './step';
-import { VoidCallback } from '../common_types';
+import { VoidCallback, HooksCallback, RunParamaters } from '../common_types';
 
 export type JourneyOptions = {
   name: string;
   id?: string;
 };
 
 type HookType = 'before' | 'after';
-export type Hooks = Record<HookType, Array<VoidCallback>>;
-
+export type Hooks = Record<HookType, Array<HooksCallback>>;
 export type JourneyCallback = (options: {
   page: Page;
   context: BrowserContext;
   browser: Browser;
   client: CDPSession;
-  params: Record<string, any>;
+  params: RunParamaters;
```

Comment:
Typo, parameters not parameters, also, can we call the arg runParams?

Reply:
Sure, lets change it to runParams

Reply:
I really have to change my keyboard :P, fixing it.